raygun4go.go

```diff
@@ -175,10 +177,22 @@ func (c *Client) createPost(err error, stack stackTrace) postData {
 	return newPostData(c.context, err, stack)
 }
 
-// CreateError is a simple wrapper to manually post messages (errors) to raygun
+// Manually send a new error with the given message to Raygun. This will use the current execution stacktrace.
 func (c *Client) CreateError(message string) error {
 	err := errors.New(message)
 	post := c.createPost(err, currentStack())
+	
+	return c.submit(post)
+}
+
+// Manually send the given github.com/go-errors/errors.Error to Raygun. This will use the message and stacktrace of the given Error.
+func (c *Client) SendError(error *goerrors.Error) error {
```

Comment:
You should probably make this just take an `error` and internally use conditional type checking to handle cases with the stack and cases which are just an error (glorified string) https://tour.golang.org/methods/15

Reply:
Thanks, I will do that